presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt

```diff
@@ -1,126 +1,286 @@
+
+        set simplified_master_multiplier = 130
+        set simplified_i_gain = 85
+        set simplified_d_gain = 110
+        set simplified_pi_gain = 110
+        set simplified_dmax_gain = 0
+        set simplified_feedforward_gain = 95
+        set simplified_pitch_pi_gain = 105
+
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED):6s Lipo with HD Camera (1750 - 1950kv motors)
+
+        set simplified_master_multiplier = 115
+        set simplified_i_gain = 85
+        set simplified_d_gain = 110
+        set simplified_pi_gain = 110
+        set simplified_dmax_gain = 0
+        set simplified_feedforward_gain = 95
+        set simplified_pitch_pi_gain = 115
+
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED):6s Lipo no HD Camera (1750 - 1950kv motors)
+
+        set simplified_master_multiplier = 115
+        set simplified_i_gain = 85
+        set simplified_d_gain = 110
+        set simplified_pi_gain = 110
+        set simplified_dmax_gain = 0
+        set simplified_feedforward_gain = 95
+        set simplified_pitch_pi_gain = 105
+
+    #$ OPTION END
+
+#$ OPTION_GROUP END
+
+# ---- Filters Base Setup -------
+
+#$ INCLUDE: presets/4.3/filters/defaults.txt
 
 set dshot_bidir = OFF
-set simplified_gyro_filter = OFF
-set gyro_lpf2_static_hz = 1000
-set gyro_lpf1_dyn_min_hz = 300
-set gyro_lpf1_dyn_max_hz = 600
-set dyn_notch_count = 3
-set dyn_notch_q = 300
-set dyn_notch_min_hz = 100
+set simplified_gyro_filter_multiplier = 120
+set gyro_lpf1_static_hz = 0
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set dyn_notch_count = 5
+set dyn_notch_q = 400
+set dyn_notch_min_hz = 90
 set dyn_notch_max_hz = 800
 
-# -- Dterm filtering --
-
-set simplified_dterm_filter = OFF
+set simplified_dterm_filter_multiplier = 140
 set dterm_lpf1_dyn_min_hz = 0
 set dterm_lpf1_dyn_max_hz = 0
 set dterm_lpf1_dyn_expo = 0
-set dterm_lpf1_static_hz = 80
-set dterm_lpf2_static_hz = 140
 
-# -- Antigravity --
+#$ OPTION_GROUP BEGIN: Filters (default to MPU6000/BMI/ICM206xx Gyro)
 
-set anti_gravity_gain = 4000
+    #$ OPTION BEGIN (CHECKED): RPM filtering (Uncheck for Dynamic Notch Filter Setup)
+    #$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
 
-#$ OPTION BEGIN (CHECKED): RPM filtering
+        set dshot_bidir = ON
+        set motor_pwm_protocol = DSHOT600
+        set rpm_filter_harmonics = 3
+        set rpm_filter_q = 1000
+        set dyn_notch_count = 2
+        set rpm_filter_fade_range_hz = 100
 
-set motor_pwm_protocol = DSHOT600
-set dshot_bidir = ON
-set simplified_gyro_filter = OFF
-set rpm_filter_harmonics = 1
-set gyro_lpf2_static_hz = 1000
-set gyro_lpf1_dyn_min_hz = 300
-set gyro_lpf1_dyn_max_hz = 600
-set gyro_lpf1_dyn_expo = 8
-set dyn_notch_count = 2
-set dyn_notch_q = 400
-set dyn_notch_min_hz = 100
-set dyn_notch_max_hz = 800
+    #$ OPTION END 
+
+    #$ OPTION BEGIN (UNCHECKED): ICM42688 Gyro setup
+
+        set gyro_hardware_lpf = OPTION_2
```

Comment:
why do you need that? I thought it does not make a difference in 4.3/4.4

Reply:
to be honest I'm not sure if it does or doesn't. I swear I can feel a difference and the lack of noise change might be due to another downstream noise issue - the filter could still partly be working so I put it in the place I would want it in case it is/it is fixed.